packages/datadog_flutter_plugin/integration_test_app/lib/auto_integration_scenarios/rum_auto_instrumentation_second_screen.dart

```diff
@@ -66,7 +66,7 @@ class _RumAutoInstrumentationSecondScreenState
       appBar: AppBar(
         title: const Text('Secondary Screen'),
       ),
-      body: FutureBuilder(
+      body: FutureBuilder<void>(
```

Comment:
I think it makes more sense to specify this up on the _loadingFuture instead of on the Builder.